.gitignore

```diff
@@ -1,2 +1,5 @@
+@@ -1,2 +1 @@
 *.agdai
 MAlonzo/**
+# test.md
```

Comment:
Please uncomment `test,md`, and `node_modules` from `.gitignore`

Reply:
uncommented test.md and node_modules